shell/platform/android/android_context_gl.cc

```diff
@@ -3,9 +3,15 @@
 // found in the LICENSE file.
 
 #include "flutter/shell/platform/android/android_context_gl.h"
-
+#include "third_party/skia/include/gpu/GrContext.h"
+#include "third_party/skia/include/gpu/gl/GrGLInterface.h"
 #include <utility>
 
+// TODO: A newer version of eglext.h should recognize these extensions?
+// #include <EGL/eglext.h>
+#define EGL_GL_COLORSPACE_KHR 0x309D
```

Comment:
This seems to be dependent on `EGL_EGLEXT_VERSION`. How about we include `EGL/eglext.h` unconditionally. Then check if these two macros are defined. And, if not, define them to these known values.